### PR TITLE
 updated packages JasmineJS matchers

### PR DESCRIPTION
**Pull request by AlexanderTunick**

### CODE REVIEW

**Comment by Xotabu4** [comment]
@Alextunchii thanks for your PR! 
Can you please kick off from pull request some meta files:
- .DS_Store
- ./idea/ folder

File: generators/app/templates/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# <%=testProjectName%>
+# myproject
```
Comment (Xotabu4) [inline generators/app/templates/README.md]: Please revert this. Yeoman Generator will put project name here, depending on what was passed as parameters

File: generators/app/templates/config/protractor.conf.ts [hunk 2]
```diff
@@ -10,11 +10,12 @@ let conf: Config = {
     '../specs/**/*.spec.js',
     '../specs/*.spec.js'
   ],
-  baseUrl: '<%=baseUrl%>',
+  baseUrl: 'http://build-a-list-test.com/',
```
Comment (Xotabu4) [inline generators/app/templates/config/protractor.conf.ts]: Please revert this. Yeoman Generator will put project url here, depending on what was passed as parameters

File: generators/app/templates/package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "<%=testProjectName%>",
+  "name": "myproject",
```
Comment (Xotabu4) [inline generators/app/templates/package.json]: Please revert this. Yeoman Generator will put project name here, depending on what was passed as parameters

File: generators/app/templates/page_objects/home.page.ts [hunk 4]
```diff
@@ -1,6 +0,0 @@
-import { browser, $} from 'protractor'
```
Comment (Xotabu4) [inline generators/app/templates/page_objects/home.page.ts]: Keep this file in original project. This is just example how to built pageobjects

**Review by Xotabu4 — CHANGES_REQUESTED** [review]

**Comment by Xotabu4** [comment]
@Alextunchii please make sure unittests are passing
https://travis-ci.org/Xotabu4/generator-modern-protractor/builds/311375263?utm_source=github_status&utm_medium=notification

**Comment by Xotabu4** [comment]
@Alextunchii Basically in your PR only need to be updated 
package.json  - update versions, kick out types/node (or better to put correct version there automatically)
protractor.config.ts - update import of matchers
homepage.spec.ts - remove this:
```
// Unfortunatelly had to loose types here. 
// Extending jasmine matchers does not work well - 
// https://medium.com/@cwmrowe/making-jasmine-and-typescript-play-nicely-c2f4bef1830a
declare let expect:any
```

**Comment by AlexanderTunick** [comment]
@Xotabu4 Thanks for information

I guess I've done almost everything, is it ok? or something else?

last changes, i dunno why haven't passed

File: generators/app/templates/config/protractor.conf.ts [hunk 5]
```diff
@@ -10,11 +10,14 @@ let conf: Config = {
     '../specs/**/*.spec.js',
     '../specs/*.spec.js'
   ],
-  baseUrl: '<%=baseUrl%>',
+
+  baseUrl: 'http://website.com/',
```
Comment (Xotabu4) [inline generators/app/templates/config/protractor.conf.ts]: please revert back to template string

File: generators/app/templates/config/protractor.conf.ts [hunk 6]
```diff
@@ -29,13 +32,14 @@ let conf: Config = {
       consolidateAll: true
     }
     jasmine.getEnv().addReporter(new JUnitXmlReporter(junit_reporter_options))
+      const protractorMatchers = require('jasmine-protractor-matchers');
```
Comment (Xotabu4) [inline generators/app/templates/config/protractor.conf.ts]: please switch to `import syntax`

**Review by Xotabu4 — CHANGES_REQUESTED** [review]

**Comment by Xotabu4** [comment]
@Alextunchii 
- .idea folder files are still there
- package.lock files not needed
- unit tests are failing
- Please do not rename home.page to login page
- please remove generators/app/templates/.gitignore
- also see comments

**Comment by AlexanderTunick** [comment]
one more attempt

**Comment by AlexanderTunick** [comment]
@Xotabu4 done everything you told me, but still, CI is not passing test

**Pull request closed without merging**